src/index.html

```diff
@@ -8,7 +8,10 @@
     <title>Colored blocks</title>
     <link rel="stylesheet" href="style.css">
   </head>
-  <body>
-    <h1>Colored blocks</h1>
+  <body >
+    <!-- <h1>Colored blocks</h1> -->
```

Comment:
if you don't use it, feel free to remove this part ;)

Reply:
fixed